manage_breast_screening/record_a_mammogram/tests/system/test_user_submits_cannot_go_ahead_form.py

```diff
@@ -43,9 +43,16 @@ def when_i_submit_the_form(self):
         self.page.get_by_role("button", name="Continue").click()
 
     def then_i_should_see_validation_errors(self):
-        expect(
-            self.page.locator("#stopped_reasons-error")
-        ).to_have_text(re.compile("A reason for why this appointment cannot continue must be provided"))
+        self.expect_validation_error(
+            id="stopped_reasons-error",
```

Comment:
I wonder if we can remove the need for this ID, and instead validate things based on
1. the error text
2. the label of the field we expect to have focus

I think that would align the tests more with what end users expect from the interaction.